app/views/views.py

```diff
@@ -0,0 +1,69 @@
+
+# your_app/views.py
+from flask import jsonify
+from bson import ObjectId
+from ..schema.schema import User
+from .. import mongo
+from ..route.route import main_blueprint,request
+
+@main_blueprint.route('/', methods=["GET"])
+def home():
+    return jsonify({"msg":" Successfully run"})
+
+# @main_blueprint.route('/come', methods=["GET"])
+# def act():                                                           
+#     return jsonify({"msgg":"Done!"})
+
+
+@main_blueprint.route('/users', methods=["GET","POST"])
+def get_users():
+    if request.method == "GET":
+        #Here I am handling Get request 
```

Comment:
Remove unnecessary comments, Add comments for a functionality not a small if condition